### DISCUSSION TITLE
Manually combine two drives?

### DISCUSSION THREAD

**Discussion by Str1atum** [teslamate-org/teslamate#2213] [Ideas]
After just switching to "P" for some seconds the drive is splitted into two separate instances – is there a way to manually combine two drives again?
Or maybe as a feature suggestion?

**Comment by Kosta-Github** [comment]
Or maybe combine drives automatically that have a park/pause state in between of less than 3(?) minutes…

**Comment by Hellraiser8** [comment]
Would love that as well
